src/node.cc

```diff
@@ -2600,6 +2608,7 @@ static void GetLinkedBinding(const FunctionCallbackInfo<Value>& args) {
 
 static void ProcessTitleGetter(Local<Name> property,
                                const PropertyCallbackInfo<Value>& info) {
+  Mutex::ScopedLock lock(process_title_mutex);
```

Comment:
Not necessary, `uv_get_process_title()` and `uv_set_process_title()` already use locks internally.

(Perhaps `uv_os_getenv()`, `uv_os_setenv()` and `uv_os_unsetenv()` should do the same.)

Reply:
Done, thanks for pointing this out.